autogen/code_utils.py

```diff
@@ -8,7 +8,7 @@
 import time
```

Comment:
is there a plan to move code_utils.py implementation to coding dir?

Reply:
Right. Overtime we can deprecate the `code_utils` and have a library of supported executors contributed by the community.